lib/cn/node_split.c

```diff
@@ -134,7 +134,7 @@ find_inflection_key(
     iters = buf;
     srcs = buf + num_kvsets * sizeof(*iters);
 
-    list_for_each_entry(le, &node->tn_kvset_list, le_link) {
+    list_for_each_entry(le, &tnode->tn_kvset_list, le_link) {
```

Comment:
Everywhere else we call it "node" or "tn"....

Reply:
Nabeel renamed the other function to use tnode, so I just changed it here to be consistent. I can change both to tn though. I assume he changed it to be less ambiguous with WBT node